setup.py

```diff
@@ -1,53 +1,7 @@
 import os
-from setuptools import find_packages, setup
-
-with open(os.path.join(os.path.dirname(__file__), "README.rst")) as readme:
-    README = readme.read()
-
-description = (
-    "Run any Django management command on an AWS Elastic Container Service"
-    "(ECS) cluster."
-)
+from setuptools import setup
 
 # allow setup.py to be run from any path
 os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
 
-setup(
-    name="django-ecsmanage",
-    use_scm_version=True,
-    packages=find_packages(),
-    include_package_data=True,
-    license="Apache License 2.0",
-    description=description,
-    long_description=README,
-    url="https://github.com/azavea/django-ecsmanage/",
-    author="Azavea, Inc.",
-    author_email="systems@azavea.com",
-    install_requires=[
-        "Django>=1.11,<3.1",
```

Comment:
The upper bound of this was relaxed in `setup.cfg` to allow building the Django `master` branch. (azavea/django-amazon-ses#63)